apps/frontend/src/app/2_molecules/AmountRenderer/AmountRenderer.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC, useCallback, useMemo } from 'react';
+
+import { Icon, IconNames, Tooltip, TooltipTrigger } from '@sovryn/ui';
+
+import { formatValue } from '../../../utils/math';
+
+type AmountRendererProps = {
+  value: string | number;
+  precision?: number;
+  className?: string;
+  sufix?: string;
+  prefix?: string;
+  dataAttribute?: string;
+};
+
+export const AmountRenderer: FC<AmountRendererProps> = ({
+  value,
+  className,
+  precision = 4,
+  sufix = '',
+  prefix = '',
+  dataAttribute,
+}) => {
+  const copyAddress = useCallback(async () => {
+    await navigator.clipboard.writeText(String(value));
+    alert('Address was copied to clipboard.');
```

Comment:
We want to show a `Notification` (look at `EmailNotificationSettingsDialogComponent` for instance) instead of a JS alert.